src/main/kotlin/scene/PlayScene.kt

```diff
@@ -0,0 +1,62 @@
+package scene
+
+import engine.EnvironmentManager
+import model.Car
+import model.ScoreBoard
+import port.OutputPainter
+import utils.RandomGenerator
+
+class PlayScene(
+    private val outputPainter: OutputPainter,
+    private val scoreBoard: ScoreBoard,
+    private val randomGenerator: RandomGenerator,
+) : Scene {
+
+    private var currentRound = 1
+    private var totalRound = 0
+    private var cars: List<Car> = emptyList()
```

Comment:
```suggestion
    private val cars = mutableListOf<Car>()
```
컬렉션 자체에서 add, remove등 데이터 조작이 가능하기에, 컬렉션은 var을 사용하는 것을 지양해요

Reply:
Car를 불변 객체로 만들면서 항상 리스트를 갱신하는 방식으로 진행한 결과인데 혹시 일급 객체를 만들어 접근하는 방법은 괜찮을까요?

Reply:
일급 객체에 대해서 알고 계시는군요 👍 
일급 객체로 만든다면 가장 좋을 것 같아요 💯